engine/src/flutter/lib/web_ui/README.md

```diff
@@ -250,6 +250,36 @@ Instead, we update this file manually once in a while.
 `canvaskit_lock.yaml` locks the version of CanvasKit for tests and production
 use.
 
+### Debugging the Web Engine
+
+Build the Flutter Web engine locally:
+
+```
+felt build
+```
+
+Run a Flutter app in debug mode using your locally built Web Engine artifacts:
+
+* **Option 1**: Launch a Chrome window from the command line.
+   ```
+   flutter run --local-web-sdk=wasm_release --debug -d chrome
+   ```
+   Exiting `flutter run` will close the app's Chrome window.
+* **Option 2**: Launch a web server on port `8080`:
+  ```
+  flutter run --local-web-sdk=wasm_release --debug -d web-server --web-port 8080
+  ```
+  To see your Flutter app, navigate your browser to http://localhost:8080.
+  This option is useful if you want to keep your browser window when you
```

Comment:
This option is also useful for debugging non-Chromium browsers that are not supported by `flutter run`, such as Firefox and Safari.

Reply:
Ah interesting, will add!

Reply:
Added!